scripts/kvm-test.py

```diff
@@ -226,14 +237,20 @@ def mounter(src, dest):
 
 
 def livefs_edit(ctx, *args):
-    run(['sudo', 'PYTHONPATH=$LIVEFS_EDITOR', 'python3', '-m', 'livefs_edit',
-           ctx.baseiso, ctx.iso, *args])
+    livefs_editor = os.environ['LIVEFS_EDITOR']
+    run(['sudo', f'PYTHONPATH={livefs_editor}', 'python3', '-m', 'livefs_edit',
+         ctx.baseiso, ctx.iso, *args])
 
 
 def build(ctx):
     remove_if_exists(ctx.iso)
     project = os.path.basename(os.getcwd())
 
+    http_proxy = os.environ.get('DEBOOTSTRAP_PROXY')
+    snapargs = '--debug'
+    if http_proxy:
```

Comment:
minor: maybe swap this instruction with the one above so it's not interleaved with the proxy management?

(you can disregard this comment, it's nothing important)

```suggestion
    snapargs = '--debug'
    http_proxy = os.environ.get('DEBOOTSTRAP_PROXY')
    if http_proxy:
```